app/src/main/java/com/humberto/tasky/core/presentation/designsystem/components/TaskyCalendarHeader.kt

```diff
@@ -65,14 +65,17 @@ fun TaskyCalendarHeader(
             modifier = Modifier,
             horizontalArrangement = Arrangement.spacedBy(16.dp)
         ) {
-            items(daysList) { day ->
+            items(daysList, key = {
+                day -> day
+            }) { day ->
```

Comment:
Nitpick - when you want to use the entire model as a key, I find the following more reasable:

```
items(daysList, key = { it }) { day ->
  // ...
}
```